x-pack/plugins/canvas/public/components/workpad_loader/workpad_loader.js

```diff
@@ -173,7 +173,8 @@ export class WorkpadLoader extends React.PureComponent {
         sortable: true,
         dataType: 'string',
         render: (name, workpad) => {
-          const workpadName = workpad.name.length ? workpad.name : <em>{workpad.id}</em>;
+          let workpadName = name.length ? name : <em>{workpad.id}</em>;
+          workpadName = workpad.id === loadedWorkpad ? <strong>{workpadName}</strong> : workpadName;
```

Comment:
nit: hoisting this into a helper function would allow you to avoid the re-assignment and just return the display name. 

```
const getDisplayName = (name, workpad, loadedWorkpad) => {
  const workpadName = name.length ? name : <em>{workpad.id}</em>;
  return workpad.id === loadedWorkpad ? <strong>{workpadName}</strong> : workpadName;
}
```